### PR TITLE
Remove mention of C: drive in Windows path

### LINKED ISSUE #6179: set PYTHONPATH info is not right/up to date
QGIS 3.10 on Windows

## Description
Making custom python application need the "import qgis.core" python command, which returns "ModuleNotFoundError: No module named 'qgis'"
Documentation says, "on Windows: set PYTHONPATH=c:\<qgispath>\python" (which seems suspicious by itself, there is no such file after installing qgis-ltr 3.10)
After adding it to the PYTHON PATH, the ModuleNotFoundError is still exactly the same.

What is/are the correct uri to add to the PYTHONPATH?

Page URL:
https://docs.qgis.org/3.10/en/docs/pyqgis_developer_cookbook/intro.html#running-custom-applications

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #6179

### CODE REVIEW

**Review by havatv — APPROVED** [review]

**Review by havatv — APPROVED** [review]
Looks good to me.

File: docs/pyqgis_developer_cookbook/intro.rst [hunk 1]
```diff
@@ -299,7 +299,7 @@ the following commands, ``<qgispath>`` should be replaced with your actual
 QGIS installation path:
```
Comment (havatv) [inline docs/pyqgis_developer_cookbook/intro.rst:299]:
```suggestion
QGIS installation path(e.g. `C:...... for an OSGeo4W install:
```
If someone knows the install psth, please prove it (or change to the stand alone one, orthe Mac or Ubunu one)

**Review by havatv — COMMENTED** [review]
Looks goog.Onesuggestion /question.

**Comment by DelazJ** [comment]
Closing for #6306

**Pull request closed without merging**